test/inference/gibbs_conditional.jl

```diff
@@ -0,0 +1,122 @@
+using Random, Turing, Test
+using Clustering
+
+dir = splitdir(splitdir(pathof(Turing))[1])[1]
+include(dir*"/test/test_utils/AllUtils.jl")
+
+
+@turing_testset "gibbs conditionals" begin
+    let N = 1000,
+        (α_0, θ_0) = (2.0, inv(3.0)),
+        λ_true = rand(Gamma(α_0, θ_0)),
+        σ_true = sqrt(1 / λ_true),
+        m_true = rand(Normal(0, σ_true)),
+        x = rand(Normal(m_true, σ_true), N)
+        
+        @model function inverse_gdemo(x)
+            λ ~ Gamma(α_0, θ_0)
+            m ~ Normal(0, sqrt(1 / λ))
+            @. x ~ $(Normal(m, sqrt(1 / λ)))
```

Comment:
A bit simpler would be
```suggestion
            σ2 ~ InverseGamma(α_0, θ_0)
            σ = sqrt(σ2)
            m ~ Normal(0, σ)
            @. x ~ $(Normal(m, σ))
```